packages/map-template/src/components/Map/Map.jsx

```diff
@@ -27,9 +27,10 @@ const localStorageKeyForVenue = 'MI-MAP-TEMPLATE-LAST-VENUE';
  * @param {function} props.onVenueChangedOnMap - Function that is run when the map bounds was changed due to fitting to a venue.
  * @param {function} props.onUserPosition - Function that is run when (if) the user position updates. Sends position as payload.
  * @param {array} props.filteredLocationIds - Array of IDs of the filtered locations.
+ * @param {function} props.onSelectedMapType - Function that is run when the map type is changed.
```

Comment:
```suggestion
 * @param {function} props.onMapTypeChanged - Function that is run when the map type is changed.
```
? More in alignment with the description.